src/intree.js

```diff
@@ -49,6 +49,7 @@ function completeInTreeConfig(config, payload) {
           GITHUB_HEAD_BRANCH: payload.details['event.head.repo.branch'],
           GITHUB_HEAD_REF: payload.details['event.ref'],
           GITHUB_HEAD_USER_EMAIL: payload.details['event.head.user.email'],
+          GITHUB_INSTALLATION_ID: payload.installationID,
```

Comment:
Do you know of a case where this will be useful?  Installations are just parts of an integration, and the created task is not related to the integration.  In other words, without the PEM, I think this value is not useful.  So there's probably no point in sending it to the task.